app/assets/javascripts/ui.js

```diff
@@ -135,8 +135,11 @@ window.attachTooltipListener = () => {
       const placement = [...trigger.classList].find(c => c.startsWith("tooltipped--"))?.split("--")[1] || "n";
       const offset = 5;
 
+      const label = trigger.getAttribute("aria-label");
```

Comment:
Should this be:

```suggestion
      const label = trigger.getAttribute("aria-label").trim();
```

And then just:

```
if (!label) return;
```